testsuite/scripts/samples/test_electrophoresis.py

```diff
@@ -19,8 +19,28 @@
 import importlib_wrapper
 import numpy as np
 
+
+def set_seed(code):
+    np_seed = "np.random.seed()"
+    assert np_seed in code
+    return code.replace(np_seed, "np.random.seed(42)")
+
+
+def set_p3m_params(code):
+    p3m = "electrostatics.P3M(prefactor=1.0, accuracy=1e-2)"
+    assert p3m in code
+    return code.replace(
+        p3m, "electrostatics.P3M(prefactor=1, mesh=[16, 16, 16], cao=7, accuracy=1e-2, r_cut=3.7781e-02, tune=False)")
```

Comment:
```suggestion
        p3m, "electrostatics.P3M(prefactor=1, mesh=[16, 16, 16], cao=1, accuracy=1e-2, r_cut=3.778125, alpha=0.19510154, tune=False)")
```
you accidentally used `r_cut_iL` instead of `r_cut`

Reply:
ah you are right